examples/interactive_play.py

```diff
@@ -69,13 +69,17 @@
     observations_to_image,
     overlay_frame,
 )
+from habitat_baselines.articulated_agent_controllers import (
+    HumanoidRearrangeController,
+)
 from habitat_sim.utils import viz_utils as vut
 
 try:
     import pygame
 except ImportError:
     pygame = None
 
+WALK_POSE_PATH = "data/humanoid_data/walking_motion_processed.pkl"
```

Comment:
Can you change this to be a command line argument instead since it seems like something the user would want to change?